client/boot/index.js

```diff
@@ -239,6 +239,11 @@ function reduxStoreReady( reduxStore ) {
 				user_id: user.get().ID,
 				calypso_env: config( 'env_id' )
 			} );
+
+			if ( global.window && window.app && window.app.commitChecksum ) {
```

Comment:
Wait. We actually do get`global`, [from webpack](https://github.com/Automattic/wp-calypso/blob/master/webpack.config.js#L77).